### PR TITLE
add pangenome browser prep

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
I'll move the other tools into the shared library as well

**Review by tetron — COMMENTED** [review]
Wasn't there a request that data be organized into a particular directory structure?

> Unless this functions differently, but I believe Schematize relies on the bin2file to access the correct chunks
There should also be multiple chunk sizes in separate folders according to size
so like readsMergedDedup/4000/chunk<0-n>...

I only see File output which wouldn't produce the desired directory structure?

**Comment by mandosoft** [comment]
The component_segmentation should produce an output for Schematize like this:
readsMergedDedup.seg>
    1>chunksw1.json...
    4>chunksw4.json...
    16...
    64
    256
    1000
    4000
    16000
    bin2file.json

**Comment by mandosoft** [comment]
bin2file is used by Schematize to index the public data directory and access chunks according to selected zoom level by the user

**Comment by mr-c** [comment]
@tetron @mandosoft  I've upgraded this to preserve the directory structure of `segmentation.py`'s output

File: workflows/pangenome-generate/pangenome-generate.cwl [hunk 1]
```diff
@@ -66,3 +97,29 @@ steps:
       originalLabels: relabel/originalLabels
     out: [merged]
     run: merge-metadata.cwl
+  bin_paths:
+    run: ../tools/odgi/odgi_bin.cwl
+    in:
+      sparse_graph_index: buildGraph/odgiGraph
+      bin_width: bin_widths
+    scatter: bin_width
+    out: [ bins, pangenome_sequence ]
```
Comment (mr-c) [inline workflows/pangenome-generate/pangenome-generate.cwl:106]: @tetron , these scatter steps are really fast, so I guess we could add a `arv:RunInSingleContainer` hint?

**Pull request merged**